test/testutil/tests.c

```diff
@@ -293,7 +164,7 @@ DEFINE_COMPARISONS(char, char, "%c")
 DEFINE_COMPARISONS(unsigned char, uchar, "%u")
 DEFINE_COMPARISONS(long, long, "%ld")
 DEFINE_COMPARISONS(unsigned long, ulong, "%lu")
-DEFINE_COMPARISONS(size_t, size_t, "%" OSSLzu)
+DEFINE_COMPARISONS(size_t, size_t, "%zu")
```

Comment:
Nope.  The condition in testutil.h is defined locally in each test program (currently, only bioprintttest.c), so it doesn't affect the implementation.  The reason to do this is that %zu isn't necessarely available with `vsnprintf()`